sdk/include/opentelemetry/sdk/trace/batch_span_processor.h

```diff
@@ -66,8 +66,11 @@ class BatchSpanProcessor : public SpanProcessor
    * NOTE: This method is a no-op.
    *
    * @param span - The span that just started
+   * @param parent_context - The parent context of the span that just started
    */
-  void OnStart(Recordable &span) noexcept override;
+  void OnStart(Recordable &span,
+               const opentelemetry::trace::SpanContext &parent_context =
+                   opentelemetry::trace::SpanContext::GetInvalid()) noexcept override;
```

Comment:
Any reason for having a default value here? According to the spec this parameter is required, so I think we shouldn't make it easy to omit it.

Reply:
Yes right, it should be required parameter. I have modified the function prototype accordingly.